apps/judicial-system/api/src/app/modules/case/case.controller.ts

```diff
@@ -79,4 +75,16 @@ export class CaseController {
 
     return updatedCase
   }
+
+  @Get('case/:id/notifications')
+  @ApiOkResponse({ type: Notification, isArray: true })
+  getAllNotificationsById(@Param('id') id: string): Promise<Notification[]> {
+    return this.caseService.getAllNotificationsByCaseId(id)
+  }
+
+  @Post('caser/:id/notification')
```

Comment:
Is this a typo? Is this supposed to be "case/:id"?

Reply:
Yessör